docs/processes/quality-assurance/templates/test-strategy-template.md

```diff
@@ -0,0 +1,54 @@
+## **Test Strategy (Sample)**
```

Comment:
**nitpick:** Similar to suggestion above. Since this is a header, I think we don't need emphasis (ie. `**`). There is no visual difference 🤔